test/cli.ts

```diff
@@ -1,12 +1,13 @@
 import tape from 'tape';
 import spawn from 'tape-spawn';
 import * as path from 'path';
-const pkg = require('../package.json');
+import solc from '../';
 
 tape('CLI', function (t) {
   t.test('--version', function (st) {
     const spt = spawn(st, './solc.js --version');
-    spt.stdout.match(RegExp(pkg.version + '(-[^a-zA-A0-9.+]+)?(\\+[^a-zA-Z0-9.-]+)?'));
```

Comment:
This old regex actually seems very broken. Given that the string it's meant to match is something like `0.8.10+commit.fc410830.Emscripten.clang`:
- `A-A` should probably have been `A-Z`.
- The character groups should not be negated with `^`.

I think that only reason this regex "worked" is that these parts are optional (`?`) and the regex was not anchored with `^` and `$`.